package.json

```diff
@@ -63,6 +63,10 @@
 			{
 				"command": "cursorless.resumeRecording",
 				"title": "Cursorless: Resume test case recording"
+			},
+			{
+				"command": "cursorless.captureErrorTests",
```

Comment:
Prob better to use existing test case recorder command and just add a field to its [config object](https://github.com/cursorless-dev/cursorless/blob/972cab15940ab20f3dcfb7205a10e01cc76d6edb/src/testUtil/TestCaseRecorder.ts#L19-L50).  See [the docs](https://www.cursorless.org/docs/contributing/test-case-recorder/#initial-setup) for how to use these fields from talon side (step 2)